addons/binding/org.openhab.binding.wink/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="wink"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+	<bridge-type id="wink_hub_2">
+    	<label>Wink Hub 2</label>
```

Comment:
You need the bridge thing.  It is the delegate for all other thing operations.  It doesn't have any functionality other than as the delegate for attached devices.

Reply:
In your implementation the bridge delegates all calls to the handlers, right. But in the ESH/openHAB terminology you only need a bridge if it is **required** to be able to access something behind it. For example in the Philips Hue binding you need a bridge, because it has an IP address etc which then offers you to access the bulbs behind it which you would not be able to reach without the bridge.

In your scenario if you can only have one wink account and store the information to access this account inside the binding configuration there is no such need for a bridge. Usually you have one handler per (physical) thing(type), but from this you can delegate the calls to your internal implementation which deals with the pubnub protocol.

Reply:
I might not be following you.  Wink is similar to the hue bridge in that it has a zwave radio that I use to control several light switches and door locks.  It also provides a hub interface for other things that might be directly reachable, but can be manipulated via the hub interface as well.  The hub does have readable properties that I just haven't implemented, but I expect that we'll find a use for them at some point.  I disagree that it should be removed. The pubnub subscription emits events detected by the hub but doesn't control the attached devices, this is done by setting the desired_state through the wink api.  I suppose you could have multiple wink hubs, I don't think you need multiple accounts to use them, but I'm unsure.

Reply:
Thinking about it again, it makes sense to keep the `bridge`, if you move the app ID/secret/token, i.e. everything that is related to the authentication, from the binding configuration to the bridge.